pkg/test/framework/components/istio/operator.go

```diff
@@ -520,8 +520,16 @@ func installControlPlaneCluster(i *operatorComponent, cfg Config, cluster resour
 	}
 
 	if cluster.IsConfig() {
-		// there are a few tests that require special gateway setup
-		// for example: security-file-mounted-certs, security-mtlsk8sca, security-sds-ingress-k8sca
+		// there are a few tests that require special gateway setup which will cause eastwest gateway fail to start
+		// exclude these tests from installing eastwest gw for now
+		testID := i.ctx.Settings().TestID
+		excludedTests := []string{"security-file-mounted-certs", "security-mtlsk8sca", "security-sds-ingress-k8sca"}
```

Comment:
I think we should allow a suite or test pass some setting to the `istio.Setup` or in the `Environment` rather than referencing individual  tests from inside the framework.

Reply:
@stevenctl sorry i haven't had chance to address your comment - saw this already merged.  would you do the follow up PR?

Reply:
BTW, we should be able to support eastwest gw for these 3 tests if we pass the extra config when setting up the eastwest gw.